app/assets/stylesheets/css3/_appearance.scss

```diff
@@ -1,3 +1,12 @@
+@charset "UTF-8";
+
+/// Used to display an element using a platform-native styling based on the operating system's theme.
+///
+/// @example scss - Usage
+///   .element {
+///     @include appearance(none);
+///   }
+
```

Comment:
We should also have a `@require` here: `@require {mixin} prefixer`

http://sassdoc.com/annotations/#require